### PR TITLE
Removes Exploitable World/Topic Command server_hop (#51793)

### PR DESCRIPTION
**Pull request by ike709**
https://github.com/tgstation/tgstation/pull/51793

## Changelog
:cl: bobbahbrown
del: Removed exploitable world/topic command server_hop
/:cl:

### CODE REVIEW

**Review by PowerfulBacon — DISMISSED** [review]

**Comment by PowerfulBacon** [comment]
Travis cant find some config files @Crossedfall

**Comment by Crossedfall** [comment]
> Travis cant find some config files @Crossedfall

That's been the case for a long time. It doesn't effect the tests. You're welcome to add them if it bothers you though.

File: code/datums/world_topic.dm [hunk 1]
```diff
@@ -150,11 +139,8 @@
 	.["players"] = GLOB.clients.len
 	.["revision"] = GLOB.revdata.commit
 	.["revision_date"] = GLOB.revdata.date
+	.["hub"] = GLOB.hub_visibility
 
-	var/client_num = 0
-	for(var/client/C in GLOB.clients)
-		.["client[client_num]"] = C.key
-		client_num++
```
Comment (Crossedfall) [inline code/datums/world_topic.dm]: Any reason why this is being removed?

Comment (ike709) [reply code/datums/world_topic.dm]: because I cherrypicked at 1am

**Pull request merged**